packages/services/api/src/modules/integrations/providers/slack-integration-manager.ts

```diff
@@ -43,10 +45,27 @@ export class SlackIntegrationManager {
       },
     });
     this.logger.debug('Updating organization');
-    await this.storage.addSlackIntegration({
+    const result = await this.storage.addSlackIntegration({
       organizationId: input.organizationId,
       token: this.crypto.encrypt(input.token),
     });
+
+    const currentUser = await this.session.getViewer();
+    await this.auditLog.record({
+      eventType: 'ORGANIZATION_UPDATED_INTEGRATION',
+      integrationId: input.organizationId,
+      updatedFields: JSON.stringify({
+        slackIntegrationStatus: 'DISABLED',
+      }),
```

Comment:
TODO @TuvalSimha, I don't think we need this object here if only the status can change.